### PR TITLE
Fix a few typos

### PR DESCRIPTION
**Pull request by twolife**
Reported by Lintian when packaging the engine for Debian; it shouldn't hurt

### CODE REVIEW

File: fred2/campaigntreeview.cpp [hunk 1]
```diff
@@ -118,7 +118,7 @@ void campaign_tree_view::OnDraw(CDC* pDC)
 	Assert(r);
 
 	pDC->SelectObject(&black_pen);
-	// draw level seperators
+	// draw level separator
```
Comment (Goober5000) [inline fred2/campaigntreeview.cpp:121]: This should be plural
```suggestion
	// draw level separators
```

**Review by Goober5000 — CHANGES_REQUESTED** [review]
A lot of changes, but they all look good.  Just one small tweak needed.

File: fred2/campaigntreeview.cpp (hunk 1 above)
Comment (twolife) [reply fred2/campaigntreeview.cpp]: Ooops, good catch :-)

**Review by Goober5000 — APPROVED** [review]

**Comment by Goober5000** [comment]
@twolife We're ready to merge this, but one of the files now has conflicts due to the addition of some XSTRs.  Could you updaate the PR?

**Pull request merged**